R/report_table.R

```diff
@@ -132,19 +133,29 @@ print.report_table <- function(x, ...) {
 }
 
 
-
 #' @export
 c.report_table <- function(...) {
   x <- list(...)
 
   out <- x[[1]]
-  for(i in 2:length(x)) {
+  for (i in 2:length(x)) {
     out <- datawizard::data_join(out, x[[i]], join = "bind")
   }
   out
 }
 
 
+#' @export
+display.report_table <- function(object, ...) {
+  # fix caption
+  if (is.null(list(...)$caption)) {
+    attr(object, "no_caption") <- TRUE
+  }
+  class(object) <- c("report_table", "parameters_model", "data.frame")
```

Comment:
why not 
`class(object) <- c("parameters_model, class(object))`

Reply:
I think then `format.report_table()` is ignored, which is not what we want?

Reply:
right